docs/source_files/pixi_tomls/activation.toml

```diff
@@ -0,0 +1,29 @@
+[project]
+channels = []
+name = "activation"
+platforms = ["linux-64"]
+
+# --8<-- [start:activation]
+[activation.env]
+# Python users often set:
+PYTHONIOENCODING = "utf-8"
+PYTHONNOUSERSITE = "1"
+# R users often set:
+PIXI_R_LIBS = "$CONDA_PREFIX/lib/R/library"
+R_LIBS = "$PIXI_R_LIBS"
+R_LIBS_USER = "$PIXI_R_LIBS"
+
+
+[activation]
+# Use sh scripts on unix
+scripts = [
+  # Common in the ROS workspaces
+  "install/setup.sh",
+  # Want to add some personall scripts to the activation:
```

Comment:
```suggestion
  # Want to add some personal scripts to the activation:
```

Reply:
Sorry I was unaware you were still reviewing added it in a new PR https://github.com/prefix-dev/pixi/pull/2902